tendermint/Cargo.toml

```diff
@@ -46,9 +49,7 @@ tai64 = { version = "3", features = ["chrono"] }
 toml = { version = "0.5" }
 uuid = { version = "0.7", default-features = false }
 zeroize = { version = "0.9" }
+hex = "0.4.0"
```

Comment:
There's a hex-compatible crate already in the dependencies:

```
use subtle_encoding::hex;
```

Reply:
:heavy_check_mark: